Ports/python3/patches/ReadMe.md

```diff
@@ -15,7 +15,6 @@ As usual, make the `configure` script recognize Serenity. Also set
 `MACHDEP` (which is used for `sys.platform`) to a version-less
 `serenityos`, even when not cross-compiling.
 
-
```

Comment:
Removing those lines is a bit sad (since they are supposed to be there after the script finishes regenerating), but I suppose a revert is a revert.

Reply:
Added those back with a note :+1: